app/web/src/newhotness/StatusPanelIcon.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <Transition
+    enterActiveClass="transition-opacity duration-300"
```

Comment:
We transition in slightly faster than we do out. This makes it less jarring on the way in while making sure it is obvious when it is fading on the way out.